Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Build the dockerfile using this command:
+# docker build --build-arg HTTPS_PROXY=http://emea.proxy.ge.com -t dtr.predix.io/pwr-smallworld/tilecacher:SW521 .
+FROM node:10.16-alpine AS base-nv-tilecacher
```

Comment:
How about we put all the files we want to have in the docker image into one directory? The Dockerfile would then be significantly simpler and in general the repository would have a better structure.

Reply:
For backwards compatibility, I think it works better to leave the application in the root directory.  Moved the example config files in to a sub folder and updated the Dockerfile.